jaeger/injector/mutator/patch.go

```diff
@@ -22,6 +22,14 @@ const tpl = `[
       "value": "{{.CollectorSvcAddr}}"
     }
   },
+  {
+    "op": "add",
+    "path": "/spec/{{.ProxyPath}}/env/-",
+    "value": {
+      "name": "LINKERD2_PROXY_TRACE_PROTOCOL",
+      "value": "{{.CollectorProtocol}}"
```

Comment:
Shouldn't this be `.CollectorTraceProtocol`?

Reply:
Yes it should, good catch!